File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    var answer = [];
    quiz.forEach((value) => {
        const arr = value.split('=');
        if(eval(arr[0]) === Number(arr[1])){
            answer.push('O');
        } else {
            answer.push('X')
        }
    });
    return answer;
}

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    for q in quiz:
        left, right = q.split(' = ')
        x, op, y = left.split()
        
        x, y, right = int(x), int(y), int(right)
        val = 0
        if op == '+':
            val = x + y
        else:
            val = x - y
        answer.append('O' if val == right else 'X')
    return answer